projects/app-crm/src/assets/images/grid/propeller-logo.svg

```diff
@@ -0,0 +1 @@
+<svg id="Layer_1" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 26.8 27.24"><defs><style>.cls-1,.cls-2,.cls-3{fill-rule:evenodd;}.cls-2{opacity:0.5;}.cls-2,.cls-3{isolation:isolate;}.cls-3{opacity:0.2;}</style></defs><title>propeller-logo</title><path class="cls-1" d="M12.81,15a1.18,1.18,0,0,1,0-1.64,1.14,1.14,0,0,1,1.62,0,1.16,1.16,0,0,1,0,1.64,1.12,1.12,0,0,1-1.62,0m10.46,4.94c.08-.71.65-4.73-2.81-4.42s-4.72.44-4.87-.55a.85.85,0,0,0,0-.17,2,2,0,0,0,.07-.94,13.6,13.6,0,0,1,2.4-.29c.85.06,8.82.6,8.78-3.51s-7.32-5.82-8-5.88-4.69-.51-4.28,3,.58,4.77-.39,5l-.16,0a1.82,1.82,0,0,0-.57,0,14,14,0,0,1-.31-2.59c0-.87.59-8.94-3.46-8.9S3.86,8,3.81,8.75,3.3,13.5,6.75,13.08s4.7-.58,4.88.41a.57.57,0,0,0,0,.14,1.81,1.81,0,0,0-.07.52,15.22,15.22,0,0,1-2.73.27C8,14.34.08,13.55,0,17.66s7.15,6,7.84,6.12,4.67.65,4.36-2.85-.44-4.77.52-4.94a2,2,0,0,0,1.07.21A14.25,14.25,0,0,1,14,18.86c-.08.87-.85,8.92,3.2,9s6-7.24,6-7.95" transform="translate(0 -0.62)"/><path class="cls-2" d="M14.46,4.47s1.18-2.12,5.88-.86c-.77-1.33-6.82-2-5.88.86" transform="translate(0 -0.62)"/><path class="cls-3" d="M14.25,2.74s.92-1.6,4.33-.6c-1-1-4.34-1.53-4.33.6" transform="translate(0 -0.62)"/><path class="cls-2" d="M22.84,15.11s2.09,1.2.85,6c1.31-.78,2-6.91-.85-6" transform="translate(0 -0.62)"/><path class="cls-3" d="M24.55,14.9s1.58.93.59,4.39c.93-1.06,1.51-4.4-.59-4.39" transform="translate(0 -0.62)"/><path class="cls-2" d="M4,13.39s-2.09-1.2-.85-6c-1.31.78-2,6.91.85,6" transform="translate(0 -0.62)"/><path class="cls-3" d="M2.3,13.61s-1.58-.94-.58-4.4c-.94,1.07-1.52,4.4.58,4.4" transform="translate(0 -0.62)"/><path class="cls-2" d="M12.33,23.49s-1.25,2.08-5.9.66c.72,1.36,6.74,2.23,5.9-.66" transform="translate(0 -0.62)"/><path class="cls-3" d="M12.48,25.23s-1,1.57-4.35.45c1,1,4.29,1.68,4.35-.45" transform="translate(0 -0.62)"/></svg>
```

Comment:
Change the auto-generated svg class names, because they pollute the global styles.